composer.json

```diff
@@ -68,4 +68,4 @@
             "vendor/bin/phpcs"
         ]
     }
-}
+}
```

Comment:
Would you kindly add an new line at the end of the file please? The reason is to be safe; some older software applications won't work correctly if the last line is not terminated with a new line. Thank you.